nvflare/fuel/f3/streaming/byte_streamer.py

```diff
@@ -82,7 +83,7 @@ def send(self, channel: str, topic: str, target: str, headers: dict, stream: Str
 
     def _transmit_task(self, task: TxTask):
 
-        while not task.stop:
+        while True:
```

Comment:
How do we know if this is done?
Or this method never ends, we use task_future object to terminate this?

Reply:
When it reaches the EOS (End of Stream), the method returns at line 92.